css/cptui.scss

```diff
@@ -347,3 +354,17 @@
 #poststuff {
 	min-width: 463px;
 }
+
+@media #{$min-tablet} {
+	.cptui-badge {
+	    padding-top: 142px;
+	    height: 50px;
+	    width: 173px;
+	    color: #fafafa;
+	    font-weight: bold;
+	    font-size: 14px;
+	    text-align: center;
+	    margin: 0 -5px;
+	    background: url(../images/cptui-icon-256x256.png) no-repeat;
+	}
+}
```

Comment:
Only real issue I'm seeing at the moment is that perhaps there is a retina issue for the graphics.
![screen shot 2016-09-30 at 7 13 05 pm](https://cloud.githubusercontent.com/assets/484075/19010206/14620dc0-8742-11e6-9934-20885656b093.png) I only have a Macbook Air so no high DPI, and the image is coming in really large.